kyo-prelude/shared/src/main/scala/kyo/Abort.scala

```diff
@@ -603,4 +603,61 @@ object Abort:
 
     end literal
 
+    sealed class CanMaskAbort
```

Comment:
I can't see the purpose of this class. Maybe `MaskAbort .apply` should take it to ensure the masking is scoped?

Reply:
`MaskAbort.apply` does take it. The signature starts with: `CanMaskAbort ?=> ...`

Reply:
Oh good point, `MaskAbort.apply` should take it -- you're right

Reply:
Added